apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.ts

```diff
@@ -65,6 +65,7 @@ router.post(
       }
       const paths = getPaths(undefined, res.locals);
 
+      console.log(res.locals.course.path);
```

Comment:
```suggestion
```

Reply:
Removed locally in [9173a63](https://github.com/PrairieLearn/PrairieLearn/pull/10677/commits/9173a63860cec149d3796686882b0ec384a3dc16).